Xamarin.PropertyEditing.Mac/Controls/PointEditorControl.cs

```diff
@@ -9,77 +9,31 @@
 
 namespace Xamarin.PropertyEditing.Mac
 {
-	internal class PointEditorControl : PropertyEditorControl
+	internal class PointEditorControl : BasePointEditorControl<CGPoint>
```

Comment:
If we have multiple subclasses that serve as shells for particular implementing types (`CGPoint`), we should name them accordingly.